### ISSUE TITLE
Unable to Craft Tools with Modded Planks

### ISSUE THREAD

**Issue by SHXRKIEgg** [skylorbeck/Sentimentality3#21]
With your mod I am Unable to Craft Tools with Modded Planks such as BYG Or Biome Makeover, removing your mod makes Modded Planks be able to craft Wooden tools.
![image](https://user-images.githubusercontent.com/79353591/164194791-3bc4c753-8c12-4a94-b9c9-4830571628ad.png)

**Comment by skylorbeck** [comment]
Yeah that's because I had to change how tools are crafted to prevent the vanilla recipes from overwriting the nether wood tools I added. I'm not going to close this because I'd like to fix it but there isn't really much I can do about it.
